packages/models-library/tests/test_projects_state.py

```diff
@@ -2,12 +2,6 @@
 from models_library.projects_state import ProjectLocked, ProjectStatus
 
 
-def test_project_locked_with_missing_owner_raises():
```

Comment:
With the suggestion above, there is no need ot remove this test

Reply:
added test back :+1: